frontend/packages/core/src/Resolver/fetch.tsx

```diff
@@ -55,12 +55,17 @@ const resolveResource = async (
     [key: string]: any;
   },
   onResolve: (resultObjects: any[], failureMessages: string[]) => void,
-  onError: (message: string) => void
+  onError: (message: string) => void,
+  apiProto?: any
```

Comment:
are we planning to later expand this to take in multiple apis?

Reply:
We can modify this later when we come up with a more wholistic solution